pandas/tests/scalar/test_nat.py

```diff
@@ -435,6 +456,28 @@ def test_nat_arithmetic_td64_vector(op_name, box):
     tm.assert_equal(_ops[op_name](vec, NaT), box_nat)
 
 
+@pytest.mark.parametrize(
+    "dtype,op,out_dtype",
+    [
+        ("datetime64[ns]", operator.add, "datetime64[ns]"),
+        ("datetime64[ns]", roperator.radd, "datetime64[ns]"),
+        ("datetime64[ns]", operator.sub, "timedelta64[ns]"),
+        ("datetime64[ns]", roperator.rsub, "timedelta64[ns]"),
+        ("timedelta64[ns]", operator.add, "datetime64[ns]"),
```

Comment:
shouldn't these last for yield timedelta64[ns]?

Reply:
No.

dt64, radd --> dt64 +  Nat --> only valid if NaT is treated as td64 --> dt64
dt64, sub --> NaT - dt64 --> only valid if NaT is treated as dt64 --> td64
dt64, rsub --> dt64 - NaT --> valid either way, default to treat as dt64 --> td64
td64, add --> NaT + td64 --> valid either way, defalt to treat as dt64 --> dt64

Reply:
ok, yeah these rules are ok, but if you can put them down somewhere explicity in the code